eclass/linux-mod-r1.eclass

```diff
@@ -835,6 +835,8 @@ _modules_prepare_toolchain() {
 # If enabled in the kernel configuration, this compresses the given
 # modules using the same format.
 _modules_process_compress() {
+	! use modules-compress && return
```

Comment:
```suggestion
	use modules-compress || return
```

Reply:
fwiw please add 0 too, given `|| return` would report a failure. Aka be consistent the rest of the eclass:
```
_modules_process_sign() {
    use modules-sign || return 0
```
I know you don't like using `||` much and find it unclear, but personally find it clearer given I rely on it a lot for bash and controlling return codes. And especially when it comes to the `|| die` `|| continue` `|| return` etc.. "abort" cases.